bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "emojis-list",
   "description": "Complete list of standard emojis.",
   "homepage": "https://github.com/Kikobeats/emojis-list",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
I increment the version using [bumped](https://github.com/bumped/bumped), so please remove this.